<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactWindow</class>
 <widget class="QWidget" name="ContactWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>410</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/* VERTICAL SCROLLBAR */
 QScrollBar:vertical {
	border: none;
    background: rgba(0,0,0,0);
    width: 7px;
    margin: 15px 0 15px 0;
	border-radius: 0px;
 }

/*  HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {	
	background-color: #00a884; /* whatsapp color */
	min-height: 30px;
	border-radius: 0px;
}
QScrollBar::handle:vertical:hover{	
	background-color: #00a884;
}
QScrollBar::handle:vertical:pressed {	
	background-color: #008b6d;
}

/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
	border: none;
	background-color: #00a884;
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;
	subcontrol-position: top;
	subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {	
	background-color: #00a884;
}
QScrollBar::sub-line:vertical:pressed {	
	background-color: #008b6d;
}

/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
	border: none;
	background-color: #00a884;
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {	
	background-color: #00a884;
}
QScrollBar::add-line:vertical:pressed {	
	background-color: #008b6d;
}

/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="newChatBar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>85</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>85</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#202c33;

</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7,2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="exitContact">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color : rgba(0,0,0,0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/newPrefix/Icons/55ovjpq6bo1dpbpq69e2m1lvrq.png</normaloff>:/newPrefix/Icons/55ovjpq6bo1dpbpq69e2m1lvrq.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectContact">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgba(255,255,255,200);</string>
           </property>
           <property name="text">
            <string>Select Contact</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="createRoomButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>150</horstretch>
             <verstretch>60</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>109</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">

QPushButton{
	color : white;
    border: 2px solid ;
    border-radius: 20px;
	border-color: #00a884;
    background-color:#111b21;
    fixed-width: 80px;
	margin-left: 20px;
	margin-right:20px;
	margin-bottom:10px;
	margin-top : 10px;
}

QPushButton:hover{
	background-color: #00a884;
}
</string>
           </property>
           <property name="text">
            <string>Create</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="contactFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#111b21;
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="chatRoomFrame">
           <property name="styleSheet">
            <string notr="true">QWidget#chatRoomFrame{
	background-color: #111b21;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="searchContactsFrame">
              <property name="styleSheet">
               <string notr="true">QFrame#searchContactsFrame{
background-color : #222d34;
margin-left : 10px;
margin-right : 10px;
margin-top : 7px;
margin-bottom : 7px;
border-radius : 15px;
}
QPushButton{
background-color : rgba(0,0,0,0);
}
QLineEdit{
background-color : rgba(0,0,0,0);
color : rgba(255,255,255,200);
border:None;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="searchButton">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resource.qrc">
                   <normaloff>:/newPrefix/Icons/pngwing.com.png</normaloff>:/newPrefix/Icons/pngwing.com.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="searchLine">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="placeholderText">
                  <string>Search contacts</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,4">
               <item>
                <widget class="QPushButton" name="addContact">
                 <property name="styleSheet">
                  <string notr="true">background-color : rgba(0,0,0,0);
image: url(:/newPrefix/Daco_4818361.png);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="resource.qrc">
                   <normaloff>:/newPrefix/Icons/Daco_4818361.png</normaloff>:/newPrefix/Icons/Daco_4818361.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>30</width>
                   <height>30</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="newContact">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border:none;
color:rgba(255,255,255,200);</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="placeholderText">
                  <string>New Contact</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="textError">
                 <property name="styleSheet">
                  <string notr="true">color:#b62916;
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="styleSheet">
               <string notr="true">QScrollArea#scrollArea{
border : none;
}
QScrollBar:vertical{
border:none;
background-color : rgb(60,60,90);
width : 10px;
border-radius : 5px;
}
QScrollBar::handle:vertical{
color : rgb(0, 168, 132);
border-radius : 5px;
}
QScrollBar::handle:vertical:hover{
color : rgba(255,255,255,200);
}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="ScrollRooms">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>397</width>
                 <height>544</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">QWidget#ScrollRooms{
	background-color: #111b21;
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
